### ISSUE TITLE
Communicator:None

### ISSUE THREAD

**Issue by DriveU** [hashicorp/packer#3046]
Host OS Windows 7 Enterprise
type : vmware,
Communicator : none
Had a home grown tool which loads everything which we need through a custom answer file
Outcome :crashed

```
C:\MyDevelopment\FINAL2\SSATRIAL - None>packer build windows_7.json
Packer-7 output will be in this color.

==> Packer-7: Downloading or copying ISO
    Packer-7: Downloading or copying: file:///C:/MyDevelopment/FINAL2/SSATRIAL%20-%20None/iso/en_windows_7_ultimate_with
_sp1_x64_dvd_u_677332.iso
==> Packer-7: Creating floppy disk...
    Packer-7: Copying: ./answer_files/7/Autounattend.xml
    Packer-7: Copying: ./FILES/unattend.cmd
    Packer-7: Copying: ./FILES/upgrader.exe
==> Packer-7: Creating virtual machine disk
==> Packer-7: Building and writing VMX file
==> Packer-7: Starting virtual machine...
==> Packer-7: Waiting 2m0s for boot...
==> Packer-7: Connecting to VM via VNC
==> Packer-7: Typing the boot command over VNC...
==> Packer-7: Stopping virtual machine...
Build 'Packer-7' errored: unexpected EOF

==> Some builds didn't complete successfully and had errors:
--> Packer-7: unexpected EOF

==> Builds finished but no artifacts were created.
panic: interface conversion: interface is nil, not packer.Communicator
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: goroutine 87 [running]:
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/mitchellh/packer/builder/vmware/common.(*StepUploadTools).
Run(0xc0822bf7a0, 0x36a0c60, 0xc0822d1890, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/mitchellh/packer/bu
ilder/vmware/common/step_upload_tools.go:37 +0x2d4
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/mitchellh/multistep.(*BasicRunner).Run(0xc0822b5e40, 0x36a
0c60, 0xc0822d1890)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/mitchellh/multistep
/basic_runner.go:70 +0x337
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/mitchellh/packer/builder/vmware/iso.(*Builder).Run(0xc0821
32480, 0x36a0138, 0xc08225e0e0, 0x36a0180, 0xc082252fa0, 0x36a01b0, 0xc082236008, 0x0, 0x0, 0x0, ...)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/mitchellh/packer/bu
ilder/vmware/iso/builder.go:344 +0x18fe
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/mitchellh/packer/packer/rpc.(*BuilderServer).Run(0xc0820a1
7a0, 0x1, 0xc082252eb0, 0x0, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/mitchellh/packer/pa
cker/rpc/builder.go:93 +0x3c0
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: reflect.Value.call(0x93c7e0, 0x9b56e0, 0x13, 0xa3bdf8, 0x4, 0xc08202d
ed8, 0x3, 0x3, 0x0, 0x0, ...)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/reflect/value.go:432 +0x1211
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: reflect.Value.Call(0x93c7e0, 0x9b56e0, 0x13, 0xc08202ded8, 0x3, 0x3,
0x0, 0x0, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/reflect/value.go:300 +0xb8
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: net/rpc.(*service).call(0xc0820e4940, 0xc0820e4900, 0xc08207ad00, 0xc
082058a00, 0xc0820a1d40, 0x8a26a0, 0xc082252ea0, 0xca, 0x888ca0, 0xc082252eb0, ...)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/net/rpc/server.go:383 +0x1c8
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: created by net/rpc.(*Server).ServeCodec
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/net/rpc/server.go:477 +0x4b3
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: goroutine 1 [select]:
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/hashicorp/yamux.(*Stream).Read(0xc082156000, 0xc082160000,
 0x1000, 0x1000, 0x0, 0x0, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/yamux/str
eam.go:125 +0x3b5
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: bufio.(*Reader).fill(0xc08200eba0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/bufio/bufio.go:97 +0x1f0
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: bufio.(*Reader).ReadByte(0xc08200eba0, 0x9ad000, 0x0, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/bufio/bufio.go:229 +0x81
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/hashicorp/go-msgpack/codec.(*ioDecReader).readn1(0xc082134
930, 0x82)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpac
k/codec/decode.go:90 +0x49
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/hashicorp/go-msgpack/codec.(*msgpackDecDriver).initReadNex
t(0xc0820a18a0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpac
k/codec/msgpack.go:540 +0x48
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/hashicorp/go-msgpack/codec.(*Decoder).decode(0xc08200ec00,
 0x880e40, 0xc0820a18c0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpac
k/codec/decode.go:635 +0x77
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/hashicorp/go-msgpack/codec.(*Decoder).Decode(0xc08200ec00,
 0x880e40, 0xc0820a18c0, 0x0, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpac
k/codec/decode.go:630 +0x7b
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/hashicorp/go-msgpack/codec.(*rpcCodec).read(0xc0820e4cc0,
0x880e40, 0xc0820a18c0, 0x0, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpac
k/codec/rpc.go:88 +0xd9
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/hashicorp/go-msgpack/codec.(*goRpcCodec).ReadRequestHeader
(0xc0820e4cc0, 0xc0820a18c0, 0x0, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpac
k/codec/rpc.go:127 +0x5b
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: net/rpc.(*Server).readRequestHeader(0xc0820e4900, 0x29e63b8, 0xc0820e
4cc0, 0x0, 0x0, 0xc0820a18c0, 0xc0820d9c00, 0x0, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/net/rpc/server.go:576 +0x97
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: net/rpc.(*Server).readRequest(0xc0820e4900, 0x29e63b8, 0xc0820e4cc0,
0xc0820e4900, 0xc08207ad00, 0xc082058a00, 0x0, 0x0, 0x0, 0x0, ...)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/net/rpc/server.go:543 +0x92
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: net/rpc.(*Server).ServeCodec(0xc0820e4900, 0x29e63b8, 0xc0820e4cc0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/net/rpc/server.go:462 +0x93
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/mitchellh/packer/packer/rpc.(*Server).Serve(0xc0820a1780)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/mitchellh/packer/pa
cker/rpc/server.go:141 +0x27b
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: main.main()
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/mitchellh/packer/pl
ugin/builder-vmware-iso/main.go:14 +0xcd
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: goroutine 5 [syscall]:
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: os/signal.loop()
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/os/signal/signal_unix.go:22 +0x1f
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: created by os/signal.init.1
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/os/signal/signal_unix.go:28 +0x3e
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: goroutine 7 [chan receive]:
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/mitchellh/packer/packer/plugin.Server.func1(0xc08200e720)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/mitchellh/packer/pa
cker/plugin/server.go:92 +0x6d
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: created by github.com/mitchellh/packer/packer/plugin.Server
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/mitchellh/packer/pa
cker/plugin/server.go:96 +0xcbc
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: goroutine 8 [IO wait]:
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: net.runtime_pollWait(0x29e5ca0, 0x72, 0xc0824bc000)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/runtime/netpoll.go:157 +0x67
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: net.(*pollDesc).Wait(0xc08201f1f0, 0x72, 0x0, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/net/fd_poll_runtime.go:73 +0x41
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: net.(*ioSrv).ExecIO(0xc0820280f0, 0xc08201f0e0, 0xa3e368, 0x7, 0xb48d
60, 0x787cb0, 0x0, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/net/fd_windows.go:182 +0x177
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: net.(*netFD).Read(0xc08201f080, 0xc082136000, 0x1000, 0x1000, 0x0, 0x
0, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/net/fd_windows.go:463 +0x17e
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: net.(*conn).Read(0xc082028108, 0xc082136000, 0x1000, 0x1000, 0x0, 0x0
, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/net/net.go:172 +0xeb
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: bufio.(*Reader).fill(0xc08200e780)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/bufio/bufio.go:97 +0x1f0
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: bufio.(*Reader).Read(0xc08200e780, 0xc08207ac90, 0xc, 0xc, 0xc08207ac
92, 0x0, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/bufio/bufio.go:207 +0x267
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: io.ReadAtLeast(0x29e5f18, 0xc08200e780, 0xc08207ac90, 0xc, 0xc, 0xc,
0x0, 0x0, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/io/io.go:298 +0xed
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: io.ReadFull(0x29e5f18, 0xc08200e780, 0xc08207ac90, 0xc, 0xc, 0xc, 0x0
, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/io/io.go:316 +0x69
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/hashicorp/yamux.(*Session).recvLoop(0xc082107290, 0x0, 0x0
)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/yamux/ses
sion.go:398 +0x125
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/hashicorp/yamux.(*Session).recv(0xc082107290)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/yamux/ses
sion.go:386 +0x28
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: created by github.com/hashicorp/yamux.newSession
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/yamux/ses
sion.go:104 +0x4b8
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: goroutine 9 [select]:
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/hashicorp/yamux.(*Session).send(0xc082107290)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/yamux/ses
sion.go:348 +0x5e8
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: created by github.com/hashicorp/yamux.newSession
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/yamux/ses
sion.go:105 +0x4da
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: goroutine 10 [select]:
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/hashicorp/yamux.(*Session).keepalive(0xc082107290)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/yamux/ses
sion.go:287 +0x10e
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: created by github.com/hashicorp/yamux.newSession
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/yamux/ses
sion.go:107 +0x50d
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: goroutine 11 [select]:
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/hashicorp/yamux.(*Session).AcceptStream(0xc082107290, 0xb4
8a98, 0x0, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/yamux/ses
sion.go:191 +0x1a4
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/mitchellh/packer/packer/rpc.(*muxBroker).Run(0xc0820a1760)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/mitchellh/packer/pa
cker/rpc/mux_broker.go:126 +0x3b
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: created by github.com/mitchellh/packer/packer/rpc.NewServer
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/mitchellh/packer/pa
cker/rpc/server.go:42 +0x151
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: goroutine 88 [runnable]:
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/hashicorp/yamux.(*Stream).Read(0xc08225c000, 0xc082263000,
 0x1000, 0x1000, 0x0, 0x0, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/yamux/str
eam.go:125 +0x3b5
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: bufio.(*Reader).fill(0xc08214b4a0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/bufio/bufio.go:97 +0x1f0
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: bufio.(*Reader).ReadByte(0xc08214b4a0, 0xc0821523f0, 0x0, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/bufio/bufio.go:229 +0x81
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/hashicorp/go-msgpack/codec.(*ioDecReader).readn1(0xc082219
740, 0x74f118)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpac
k/codec/decode.go:90 +0x49
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/hashicorp/go-msgpack/codec.(*msgpackDecDriver).initReadNex
t(0xc08225e0a0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpac
k/codec/msgpack.go:540 +0x48
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/hashicorp/go-msgpack/codec.(*Decoder).decode(0xc08214b500,
 0x880ea0, 0xc0821d99e0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpac
k/codec/decode.go:635 +0x77
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/hashicorp/go-msgpack/codec.(*Decoder).Decode(0xc08214b500,
 0x880ea0, 0xc0821d99e0, 0x0, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpac
k/codec/decode.go:630 +0x7b
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/hashicorp/go-msgpack/codec.(*rpcCodec).read(0xc08214ee40,
0x880ea0, 0xc0821d99e0, 0x0, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpac
k/codec/rpc.go:88 +0xd9
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: github.com/hashicorp/go-msgpack/codec.(*goRpcCodec).ReadResponseHeade
r(0xc08214ee40, 0xc0821d99e0, 0x0, 0x0)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /Users/cbednarski/code/golang/src/github.com/hashicorp/go-msgpac
k/codec/rpc.go:123 +0x5b
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: net/rpc.(*Client).input(0xc08214b560)
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/net/rpc/client.go:109 +0xc6
2016/01/08 21:09:43 packer-builder-vmware-iso.exe: created by net/rpc.NewClientWithCodec
2016/01/08 21:09:43 packer-builder-vmware-iso.exe:      /usr/local/go/src/net/rpc/client.go:201 +0xd9
2016/01/08 21:09:43 ui error: Build 'Packer-7' errored: unexpected EOF
2016/01/08 21:09:43 Builds completed. Waiting on interrupt barrier...
2016/01/08 21:09:43 machine readable: error-count []string{"1"}
2016/01/08 21:09:43 ui error:
==> Some builds didn't complete successfully and had errors:
2016/01/08 21:09:43 machine readable: Packer-7,error []string{"unexpected EOF"}
2016/01/08 21:09:43 ui error: --> Packer-7: unexpected EOF
2016/01/08 21:09:43 ui:
==> Builds finished but no artifacts were created.
2016/01/08 21:09:43 waiting for all plugin processes to complete...
2016/01/08 21:09:43 C:\Eval\Packer\packer-builder-vmware-iso.exe: plugin process exited
!!!!!!!!!!!!!!!!!!!!!!!!!!! PACKER CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!
Packer crashed! This is always indicative of a bug within Packer.
A crash log has been placed at "crash.log" relative to your current
working directory. It would be immensely helpful if you could please
report the crash with Packer[1] so that we can fix this.
[1]: https://github.com/mitchellh/packer/issues
!!!!!!!!!!!!!!!!!!!!!!!!!!! PACKER CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

**Comment by rickard-von-essen** [comment]
Similar to #2736

**Comment by tylert** [comment]
Related to #3304.

**Comment by SwampDragons** [comment]
I can't reproduce this on a recent version of Packer, so I'm going to close. If folks are still experiencing similar problems with a modern Packer, please open a new issue :)
